tests/integration/testdata/sync/041-plugins-nested-foreign-keys/1_1/kong-consumer-groups-via-ids.yaml

```diff
@@ -0,0 +1,44 @@
+_format_version: "1.1"
+
+consumer_groups:
+  - name: group-1
+    id: 8ca63651-4068-4baa-b2b9-08dc99c296e0
+  - name: group-2
+    id: 8ca63651-4068-4baa-b2b9-08dc99c29666
+
+services:
+- name: example-service
+  port: 3200
+  protocol: http
+  host: localhost
+  routes:
+  - name: example-route-1
+    paths:
+    - /r1
+    plugins:
+      - config:
+          limit_by: consumer
+          minute: 6
+          policy: local
+        consumer_group: 8ca63651-4068-4baa-b2b9-08dc99c296e0 # group-1's id
+        enabled: true
+        name: rate-limiting
+        protocols:
+          - http
+
+routes:
```

Comment:
Why do we want multiple routes - each having a plugin referencing different consumer group?